src/Modules/AuthModule/components/Login/Login.tsx

```diff
@@ -1,6 +1,160 @@
+import logo from "../../../../assets/images/PMS 3.svg";
+import './Login.css';
+import { useForm } from "react-hook-form";
+import { NavLink, useNavigate } from "react-router-dom";
+import { FormData } from "../../../../interfaces/Auth.ts";
+import {  useState } from "react";
+import axios from "axios";
+import { toast } from 'react-toastify';
 
+
+// Function 
 export default function Login() {
-  return (
-    <div>Login</div>
-  )
+  const navigate = useNavigate();
+
+  // useForm
+  // FormData : interface (Auth.ts)
+  const {
+    register,
+    handleSubmit,
+    formState: { errors},
+  } = useForm<FormData>();
+
+
+  // Password eye 
+  const [showPassword, setShowPassword] = useState<boolean>(false);
```

Comment:
when we provide a default values, the type is inferred automatically by typescript. We can remove `<boolean>` here.